### PR TITLE
MODDATAIMP-209 - Preparation for movement mod-srs\srm to core modules

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "requires": [],
   "provides": [
     {
-      "id": "source-record-storage",
-      "version": "2.2",
+      "id": "source-record-storage-snapshots",
```
Comment (KaterynaSenchenko) [inline descriptors/ModuleDescriptor-template.json]:
```suggestion
      "id": "source-storage-snapshots",
```

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -51,16 +51,13 @@
...
-          "pathPattern": "/source-storage/snapshots/{jobExecutionId}/records",
-          "permissionsRequired": [
-            "source-storage.snapshots.records.delete"
-          ]
-        },
+        }
+      ]
+    },
+    {
+      "id": "source-record-storage-records",
```
Comment (KaterynaSenchenko) [inline descriptors/ModuleDescriptor-template.json]:
```suggestion
      "id": "source-storage-records",
```

File: descriptors/ModuleDescriptor-template.json [hunk 3]
```diff
@@ -117,22 +114,34 @@
...
           "permissionsRequired": [
-            "source-storage.populate.records"
+            "source-storage.records.get"
           ]
-        },
+        }
+      ]
+    },
+    {
+      "id": "test-records-populator",
```
Comment (KaterynaSenchenko) [inline descriptors/ModuleDescriptor-template.json]:
```suggestion
      "id": "source-storage-test-records",
```
smth like this

File: descriptors/ModuleDescriptor-template.json [hunk 4]
```diff
@@ -144,6 +153,21 @@
         }
       ]
     },
+    {
+      "id": "source-record-storage-cleanup",
```
Comment (KaterynaSenchenko) [inline descriptors/ModuleDescriptor-template.json]: maybe we should move delete records endpoint to records interface, what do you think?

File: NEWS.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 * Changed payload of endpoint for updating parsed records from ParsedRecordCollection to RecordCollection
 * Added order number to the record
 * Fixed sorting records by order number
+* Divided endpoints into more smaller interfaces.
```
Comment (KaterynaSenchenko) [inline NEWS.md]:
```suggestion
* Break down source-record-storage interface into smaller ones 
```
and pls list the new interfaces

Comment (Igor-Gorchakov) [reply NEWS.md]: Broken down ...

**Pull request merged**